composer/core/state.py

```diff
@@ -270,9 +273,10 @@ def __init__(
 
         self.profiler: Optional[Profiler] = None
 
+        self.deepspeed_config = deepspeed_config
```

Comment:
Needs docstring in the Attributes section

Reply:
done